dr_ensenso_msgs/srv/GetCameraParams.srv

```diff
@@ -0,0 +1,13 @@
+string RIGHT = Right
+string LEFT  = Left
+string MONO  = Mono
+
+uint8 CAMERA_MATRIX        = 0
+uint8 DISTORTION_MATRIX    = 1
+uint8 RECTIFICATION_MATRIX = 2
+uint8 PROJECTION_MATRIX    = 3
+
+string camera    # Specify which camera we want the parameters of.
+uint8 type       # Specify what type of parameters we want.
+---
+float64[] params # The camera parameters.
```

Comment:
Would it be possible to use `sensor_msgs/CameraInfo`?

Reply:
We could for the response, but there will be cases for which we can't fill in all fields of the CameraInfo msg. For example, the P matrix does not exist for monocular cameras. Also, if we use the switch-case implementation here 

https://github.com/fizyr/dr_ensenso/pull/6/commits/a5048b406f053e2233b2650f3558906b8209407a#diff-c8b528ef0d7770c619874d70e44e87c5R505

we only return one matrix at a time, at request. Using CameraInfo would mean that we'd need to make sure to retrieve the same matrix we requested, which I think allows for more errors (imagine asking for matrix R but you read out matrix P). Returning all matrices also doesn't make sense because sometimes they don't exist, as I mentioned above.